libs/MeshKernel/include/MeshKernel/Exceptions.hpp

```diff
@@ -50,104 +54,145 @@
 namespace meshkernel
 {
 
-    /// @brief A class for generating a variadic error message.
-    class VariadicErrorMessage final
+    /// @brief Enumeration of exit codes
+    enum ExitCode
+    {
+        Success = 0,               ///< Success
```

Comment:
That's a problematic because all API fuctions return integers. If an enum class is used the underlying integer type must be returned (cannot convert implicitly). Otherwise the return type of all API funcs must be changed.  
Thoughts?